pkg/extension/sumologicextension/extension.go

```diff
@@ -435,6 +434,10 @@ func (se *SumologicExtension) registerCollector(ctx context.Context, collectorNa
 		return credentials.CollectorCredentials{}, err
 	}
 
+	if collectorName != resp.CollectorName {
+		se.logger.Warn("Collector name already in use, registered modified name", zap.String("registered_name", resp.CollectorName))
+	}
+
```

Comment:
Nice, but not sure if we should mention `clobber` or any documentation here

Reply:
Hm, I probably should add this behaviour to the documentation to begin with. I don't want to make this log message too long.

Reply:
Updated documentation.